nmap_ftp.cc

```diff
@@ -167,9 +168,9 @@ int parse_bounce_argument(struct ftpinfo *ftp, char *url) {
     ftp->port = atoi(s);
   }
 
-  strncpy(ftp->server_name, q, MAXHOSTNAMELEN);
+  strncpy(ftp->server_name, q, FQDN_LEN);
```

Comment:
I think I've covered everything now, I just wasn't sure if they were DNS names, but now everything seems okay. Thanks again for your time!
Waiting for comments.